### ISSUE TITLE
Virtualbox investigation

### ISSUE THREAD

**Issue by wb8tyw** [ham-radio-software/D-Rats#266]
This is for investigating using Virtual Box with a prebuilt OVA on Windows 8.1-11 and Mac OS

To install Virtualbox on Microsoft windows requires downloading two packages.

https://aka.ms/vs/17/release/vc_redist.x64.exe

And then: https://www.virtualbox.org/wiki/Downloads

The Virtualbox install on Microsoft Windows will complain if Python and possibly some related python libraries are not present.  These are not needed for running VMs under Virtualbox so do not need to be installed.

In theory, once that is done, what we need to do is create an OVA file with a small Linux environment and D-Rats pre-installed.

It is likely that the end user may have to map USB serial devices or GPS dongles to the VM configuration.

This would be for running the client only.

For MAC OS, the x86_64 version would be similar to Windows install.

Currently M1 Macs are not supported for Virtualbox.
